packages/@konfik/cli/src/cli.ts

```diff
@@ -177,3 +184,144 @@ export const run = () =>
     provideCwd,
     runMain,
   )
+
+function contextLines(lines: Array<string>): Array<AnsiDoc> {
```

Comment:
Would you mind if we stick to function expressions (e.g. via `const` instead of `function`) in this repo?

Reply:
👍